tests/online_migration/sle12_online_migration/zypper_patch.pm

```diff
@@ -0,0 +1,29 @@
+# SLE12 online migration tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+
+    # fully update the system
+    zypper_call('patch --with-interactive -l');
+    zypper_call('patch --with-interactive -l', 2500);
```

Comment:
What about the scenarios that require that we test migration with a minimal patch set?

The minimal patching is automatically taken care of in the zypper migration testing you are doing, but by always doing a full update before you invalidate the testing that very important product requirement that SLE supports migration with a minimal patchset